### PR TITLE
Fix typo (boostrapper -> bootstrapper)

### PR DESCRIPTION
**Pull request by crpz1**

### CODE REVIEW

File: docs/guides/building/windows.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ This is the default setting for building the Windows Installer. It downloads the
     "bundle": {
       "windows": {
         "webviewInstallMode": {
-          "type": "downloadedBoostrapper"
+          "type": "downloadedBootstrapper"
```
Comment (FabianLars) [inline docs/guides/building/windows.md]:
```suggestion
          "type": "downloadBootstrapper"
```

File: docs/guides/building/windows.md [hunk 2]
```diff
@@ -50,8 +50,8 @@ The Windows Installer by default downloads the Webview2 bootstapper and executes
...
 | :------------------------------------------------- | :---------------------------- | :------------------------ | :----------------------------------------------------------------------------------------------------- |
-| [`downloadedBoostrapper`](#downloaded-boostrapper) | Yes                           | 0MB                       | `Default` <br /> Results in a smaller installer size, but is not recommended for Windows 7 deployment. |
-| [`embeddedBoostrapper`](#embedded-bootstrapper)    | Yes                           | ~1.8MB                    | Better support on Windows 7.                                                                           |
+| [`downloadedBootstrapper`](#downloaded-bootstrapper) | Yes                           | 0MB                       | `Default` <br /> Results in a smaller installer size, but is not recommended for Windows 7 deployment. |
+| [`embeddedBootstrapper`](#embedded-bootstrapper)    | Yes                           | ~1.8MB                    | Better support on Windows 7.                                                                           |
```
Comment (FabianLars) [inline docs/guides/building/windows.md]:
```suggestion
| [`downloadBootstrapper`](#downloaded-bootstrapper) | Yes                           | 0MB                       | `Default` <br /> Results in a smaller installer size, but is not recommended for Windows 7 deployment. |
| [`embedBootstrapper`](#embedded-bootstrapper)    | Yes                           | ~1.8MB                    | Better support on Windows 7.                                                                           |
```

**Review by FabianLars — APPROVED** [review]
thank you

**Comment by crpz1** [comment]
No problem, accidentally glazed over the wrong names there even after looking at my own config

**Pull request merged**